### PR TITLE
fix: multiple typos throughout project

### PR DESCRIPTION
**Pull request by Howie59**
### Required for all PRs:

- [x] Updated associated README.md.
- [x] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves NA

### CODE REVIEW

**Comment by Howie59** [comment]
!signed-cla

File: accumulator.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ type Accumulator interface {
 	// AddMetric adds an metric to the accumulator.
 	AddMetric(Metric)
 
-	// SetPrecision sets the timestamp rounding precision.  All metrics addeds
+	// SetPrecision sets the timestamp rounding precision.  All metrics adds
```
Comment (sspaink) [inline accumulator.go:44]:
```suggestion
	// SetPrecision sets the timestamp rounding precision.  All metrics
```

**Review by sspaink — CHANGES_REQUESTED** [review]
Thank you for resolving these typos! Just one suggested change, everything else looks good.

**Comment by powersj** [comment]
@Howie59 - how did you find all these?

File: accumulator.go (hunk 1 above)
Comment (Howie59) [reply accumulator.go]: done

**Comment by Howie59** [comment]
> @Howie59 - how did you find all these?

i read the code of the entire process

**Review by powersj — APPROVED** [review]

**Pull request merged**